### ISSUE TITLE
Double loading of json data in scripts/load_db_data.py

### ISSUE THREAD

**Issue by ofersadan85** [ohad24/encyclo-flower#14]
1. This script loads to a local mongodb and also to an external mongodb URI, it should instead only load to the URI (so we can decide if it's only local or not based on the environment variable).
2. `scripts/plant_data_17_02_2022.json` is hardcoded and will need to be changed if test data changes, it's better to setup loading of all `*.json` files in this folder

**Comment by ohad24** [comment]
> 1. This script loads to a local mongodb and also to an external mongodb URI, it should instead only load to the URI (so we can decide if it's only local or not based on the environment variable).
> 2. `scripts/plant_data_17_02_2022.json` is hardcoded and will need to be changed if test data changes, it's better to setup loading of all `*.json` files in this folder

Yeah, it's only for load plants data for development and test DBs, local (generic creds) and cloud (GH actions, private creds)

In case you want to convert it to be more programmatic please let's talk beforehand.
